custom_components/cometblue/climate.py

```diff
@@ -160,24 +165,35 @@ def max_temp(self):
 
     @property
     def hvac_mode(self):
-        if self._thermostat.manual_mode:
+        if self._thermostat.is_off:
+            return HVAC_MODE_OFF
+        elif self._thermostat.manual_mode:
             return HVAC_MODE_HEAT
         else:
             return HVAC_MODE_AUTO
 
     def set_hvac_mode(self, hvac_mode):
-        if hvac_mode == HVAC_MODE_HEAT:
-            self._thermostat.manual_mode = True
-        elif hvac_mode == HVAC_MODE_AUTO:
+        if hvac_mode == self.hvac_mode:
+            return
+
+        _LOGGER.debug("HVAC_MODE to set: {}".format(hvac_mode))
+        if self.hvac_mode == HVAC_MODE_OFF:
+            self._thermostat.is_off = False
+
+        if hvac_mode == HVAC_MODE_AUTO:
             self._thermostat.manual_mode = False
+        elif hvac_mode == HVAC_MODE_HEAT:
+            self._thermostat.manual_mode = True
+        else:
+            self._thermostat.is_off = True
 
     @property
     def hvac_modes(self):
         if self._thermostat.firmware_rev == "GEN34BLE":
             # GENIUS BLE 100 does not support manual mode
-            return (HVAC_MODE_AUTO,)
+            return HVAC_MODE_AUTO, HVAC_MODE_OFF
```

Comment:
You are right, I just checked the manual. The temperature can be adjusted manually, the device will still adhere to the programmed schedule. This behaviour should be compatible to the introduced MODE_OFF though, because off just means at set target temperature of 7.5°C, no matter if manual, temporarily or by schedule